mtags/src/main/scala-3/scala/meta/internal/mtags/MtagsEnrichments.scala

```diff
@@ -156,6 +159,71 @@ object MtagsEnrichments extends CommonMtagsEnrichments:
           )
         else false
       }
+    // Copied from dotty.tools.dotc.core.SymDenotations because it was not available for 3.0.0
```

Comment:
Great, I also think that's better than maintaining `sealedStrictDesc`. So should we just ignore tests for scala 3.0 or "adjust" the result to only scope symbols?

Reply:
It would be nice to adjust the tests so that we can easily understand Metals doesn't completely support case completion by example, but it's optional, if it's too complicated, we can just ignore for Scala3 :)

Reply:
Or maybe, for 3.0.0 we can use `sym.children`? This would also provide ehxaustive match completions, just with more general cases

Reply:
We already ignore the tests for Scala 3.0, we should only make sure it compiles.

I think it makes sense to do it as @tanishiking suggests, let's just do Nil for Scala 3.0